lambdas/shared/t4_lambda_shared/preview.py

```diff
@@ -26,7 +27,7 @@
 ELASTIC_LIMIT_BYTES = int(os.getenv('DOC_LIMIT_BYTES') or 10_000)
 ELASTIC_LIMIT_LINES = 100_000
 MAX_PREVIEW_ROWS = 1_000
-TEMP_DIR = "/tmp"
+TEMP_DIR = os.path.join('/tmp/', '')
```

Comment:
`TEMP_DIR` shouldn't be needed at all - `NamedTemporaryFile` already uses `/tmp` by default.